zilliqa/src/consensus.rs

```diff
@@ -1525,7 +1518,14 @@ impl Consensus {
         self.verify_qc_signature(&block.qc, committee.clone())?;
         if let Some(agg) = &block.agg {
             // Check if the signers of the block's aggregate QC represent the supermajority
-            self.check_quorum_in_indices(&agg.signers, &committee)?;
+
+            let mut signers = Vec::new();
+            for (index, bit) in agg.cosigned.iter().enumerate() {
+                if *bit {
+                    signers.push(index as u16);
+                }
+            }
+            self.check_quorum_in_indices(&signers, &committee)?;
```

Comment:
Can this method instead take the bitvec? From a glance we should be able to implement it just as efficiently by iterating over the bitvec directly rather than a list of indices? Then we can remove the conversion preceding this call?